spec/unit/resource/api600/synergy/network_set_spec.rb

```diff
@@ -1,9 +1,18 @@
 require 'spec_helper'
 
-RSpec.describe OneviewSDK::API500::Synergy::NetworkSet do
+RSpec.describe OneviewSDK::API600::Synergy::NetworkSet do
   include_context 'shared context'
 
-  it 'inherits from OneviewSDK::API300::Synergy::NetworkSet' do
-    expect(described_class).to be < OneviewSDK::API300::Synergy::NetworkSet
+  it 'inherits from OneviewSDK::API600::C7000::NetworkSet' do
+    expect(described_class).to be < OneviewSDK::API600::C7000::NetworkSet
+  end
+
+  describe '#initialize' do
+    context 'OneView 4.00' do
```

Comment:
I think this test case can be skipped. Because you have already inheriting from C7000 where you have covered this test case.

Reply:
They were following this in 300. So I followed the same format.